### DISCUSSION TITLE
The effect differs from the preset provided in the standalone application.

### DISCUSSION THREAD

**Discussion by ghost** [ntsc-rs/ntsc-rs#186] [General]
Does anyone know why the VHS image differs from OpenFX between the standalone application?

I use OpenFX in Vegas Pro 13

standalone
![LADA ЗАПРЕТИЛИ СНИМАТЬ! АВТОСАЛОН ЛАДА ЦЕНЫ НОЯБРЬ 2021!_ntsc](https://github.com/user-attachments/assets/46a4c56f-2ed0-4f86-b8f2-961625ea3ea5)

openfx
![Image1](https://github.com/user-attachments/assets/e4fcd9af-d39b-4f3f-8abd-c65745e3e347)

**Comment by valadaptive** [comment]
There are two reasons:
- The standalone application is downscaling the input video to 480p before processing it, but Vegas is not. [See the documentation for more information on how this affects rendering.](https://ntsc.rs/docs/openfx-plugin/)
- Some settings are set to have exponential growth instead of linear growth (like the "snow" setting), to give users finer control over them. I believe this is currently not being applied properly to presets in the OpenFX and After Effects plugins; I'll try to fix this in the next release.
